lib/ddtrace/opentracer.rb

```diff
@@ -1,40 +1,30 @@
+require 'opentracing'
+require 'opentracing/carrier'
+require 'ddtrace'
+require 'ddtrace/opentracer/carrier'
+require 'ddtrace/opentracer/tracer'
+require 'ddtrace/opentracer/span'
+require 'ddtrace/opentracer/span_context'
+require 'ddtrace/opentracer/span_context_factory'
+require 'ddtrace/opentracer/scope'
+require 'ddtrace/opentracer/scope_manager'
+require 'ddtrace/opentracer/thread_local_scope'
+require 'ddtrace/opentracer/thread_local_scope_manager'
+require 'ddtrace/opentracer/distributed_headers'
+require 'ddtrace/opentracer/propagator'
+require 'ddtrace/opentracer/text_map_propagator'
+require 'ddtrace/opentracer/binary_propagator'
+require 'ddtrace/opentracer/rack_propagator'
+require 'ddtrace/opentracer/global_tracer'
+
 module Datadog
   # Namespace for ddtrace OpenTracing implementation
   module OpenTracer
```

Comment:
Just to confirm: This file assumes that `opentracing` is available when it gets required. Is that because it never gets required automatically, e.g. it's up to customers to add `require 'ddtrace/opentracer'`?

Reply:
That's right! We never eagerly load `lib/ddtrace/opentracer.rb`: https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md#quickstart-for-opentracing